eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -334,7 +334,9 @@ def reduceJWST(eventlabel, ecf_path='./', s2_meta=None):
                                                                              deg=meta.prof_deg, n=data.intstart + n,
                                                                              isplots=meta.isplots_S3, eventdir=meta.outputdir,
                                                                              meddata=data.medapdata, hide_plots=meta.hide_plots)
-
+                #Replace NaNs with zero
```

Comment:
Zeros won't have the same impact on later steps though since things like averages will actually use those points rather than ignoring them. Wouldn't a better option would be numpy masked arrays?

Reply:
Yes, that sounds like a much better solution - how do you make a numpy masked array?

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.ma.masked_invalid.html

Reply:
Sweet thanks, fixed now!

Reply:
You should replace the comment about replacing NaNs with zeros as well

Reply:
Ah yes, fixed now.